dashboard/src/components/LoadingWrapper/LoadingWrapper.test.tsx

```diff
@@ -0,0 +1,62 @@
+import { shallow } from "enzyme";
+import * as React from "react";
+import LoadingWrapper from ".";
+import LoadingSpinner from "../LoadingSpinner";
+
+let props = {} as any;
+
+const ChildrenComponent = () => <div>Hello dad!</div>;
+
+const renderComponent = (p: any) => {
+  return shallow(
+    <LoadingWrapper {...p}>
+      <ChildrenComponent />
+    </LoadingWrapper>,
+  );
+};
+
+describe("when loaded is false", () => {
+  beforeEach(() => {
+    props = {
+      loaded: false,
+    };
+  });
+
+  it("matches the snapshot", () => {
+    const wrapper = renderComponent(props);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it("renders a spinner", () => {
+    const wrapper = renderComponent(props);
+    expect(wrapper.find(LoadingSpinner)).toExist();
```

Comment:
it is a matter of taste but I think it express more intention if you merge this test with the previous one:

```
it("renders a spinner", () => {
  const wrapper = renderComponent(props);
  expect(wrapper.find(LoadingSpinner)).toExist();
  expect(wrapper).toMatchSnapshot();
});
```

For me that means: "it should matches the snapshot but the important piece I want to ensure is that it contains a LoadingSpinner". In other case, the "matches the snapshot" is kind of meaningless: when is the snapshot wrong?

Reply:
yes but I think that is something difficult to review. I am not saying that having a general purpose snapshot is something bad I just want to have as many hints as possible to know if a change in the snapshot is something important or not. For example a change in a className vs an element disappearing. In any case, let's see how reviews go after we add the different snapshots.

Reply:
I agree @andresmgot, I do not have enough experience with snapshots, I think I have some ideal conception of how it could help us but not idea on how it will develop in practice. Let's see. I personally have mixed feelings about it, worrying that in the end they will get on the way.